feature/experimental/gribi/ate_tests/ack_in_presence_other_routes/route_ack_test.go

```diff
@@ -0,0 +1,290 @@
+
+	p3 := dut.Port(t, "port3")
+	i3 := &telemetry.Interface{Name: ygot.String(p3.Name())}
+	d.Interface(p3.Name()).Replace(t, configInterfaceDUT(i3, &dutPort3))
+}
+
+// configureATE configures port1, port2 and port3 on the ATE.
+func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
+	top := ate.Topology().New()
+
+	p1 := ate.Port(t, "port1")
+	i1 := top.AddInterface(atePort1.Name).WithPort(p1)
+	i1.IPv4().
+		WithAddress(atePort1.IPv4CIDR()).
+		WithDefaultGateway(dutPort1.IPv4)
+
+	p2 := ate.Port(t, "port2")
+	i2 := top.AddInterface(atePort2.Name).WithPort(p2)
+	i2.IPv4().
+		WithAddress(atePort2.IPv4CIDR()).
+		WithDefaultGateway(dutPort2.IPv4)
+
+	p3 := ate.Port(t, "port3")
+	i3 := top.AddInterface(atePort3.Name).WithPort(p3)
+	i3.IPv4().
+		WithAddress(atePort3.IPv4CIDR()).
+		WithDefaultGateway(dutPort3.IPv4)
+
+	return top
+}
+
+// testTraffic generates traffic flow from source network to
+// destination network via srcEndPoint to dstEndPoint and checks for
+// packet loss.
+func testTraffic(t *testing.T, ate *ondatra.ATEDevice, top *ondatra.ATETopology, srcEndPoint, dstEndPoint *ondatra.Interface) {
+	ethHeader := ondatra.NewEthernetHeader()
+	ipv4Header := ondatra.NewIPv4Header()
+	ipv4Header.DstAddressRange().
+		WithMin("203.0.113.0").
+		WithMax("203.0.113.254").
+		WithCount(250)
+
+	flow := ate.Traffic().NewFlow("Flow").
+		WithSrcEndpoints(srcEndPoint).
+		WithDstEndpoints(dstEndPoint).
+		WithHeaders(ethHeader, ipv4Header)
+
+	ate.Traffic().Start(t, flow)
+	time.Sleep(15 * time.Second)
+	ate.Traffic().Stop(t)
+
+	flowPath := ate.Telemetry().Flow(flow.Name())
+	if got := flowPath.LossPct().Get(t); got > 0 {
+		t.Errorf("LossPct for flow %s got %g, want 0", flow.Name(), got)
+	}
+}
+
+// testArgs holds the objects needed by a test case.
+type testArgs struct {
+	ctx     context.Context
+	clientA *gribi.Client
+	dut     *ondatra.DUTDevice
+	ate     *ondatra.ATEDevice
+	top     *ondatra.ATETopology
+}
+
+//Configure network instance
+func configureNetworkInstance(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	d := &telemetry.Device{}
+	ni1 := d.GetOrCreateNetworkInstance(instance)
+	ni1.Type = telemetry.E_NetworkInstanceTypes_NETWORK_INSTANCE_TYPE(*ygot.Int64(1))
+
+	dutConfPath := dut.Config().NetworkInstance(instance)
+	dutConfPath.Replace(t, ni1)
+}
+
+// configStaticRoute configures a static route.
+func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) *telemetry.NetworkInstance_Protocol_Static {
+	d := &telemetry.Device{}
+	ni1 := d.GetOrCreateNetworkInstance(instance)
+	ni1.Enabled = ygot.Bool(true)
+	ni1.Name = ygot.String(instance)
+	ni1.Description = ygot.String("Static route added by gNMI-OC")
+	static := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static-1")
+	static.Enabled = ygot.Bool(true)
+	static.Identifier = telemetry.E_PolicyTypes_INSTALL_PROTOCOL_TYPE(*ygot.Int64(10))
```

Comment:
Use telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC?

Reply:
Taken care in new push.